ui/app/registration/controllers/patientCommonController.js

```diff
@@ -223,6 +223,16 @@ angular.module('bahmni.registration')
                 }
             });
 
+            $scope.localLanguageNameIsRequired = function (nameType) {
```

Comment:
I am not sure if this is advisable anymore i.e using person attribute. OMRS already has a name model where names in different locale can be added. I would have advised that. 

Also, just because the person attribute is marked as required does not mean that value must be provided - I would have thought you enable this by some sort of configuration (which I think we already have).

Reply:
As discussed, we couldn’t see any option make the field has mandatory in the back-end so we have added